wire.go

```diff
@@ -122,13 +120,13 @@ func readTime(b *bufio.Reader, f timeFmt) (t time.Time, err error) {
 	if f == timeReadLocalDateTime {
 		format := "20060102 15:04:05"
 		if len(timeString) < len(format) {
-			return time.Now(), fmt.Errorf("ibgo: '%s' too short to be datetime format '%s'", timeString, format)
+			return t, fmt.Errorf("ibgo: '%s' too short to be datetime format '%s'", timeString, format)
```

Comment:
Any reason for this change? As  `t`  is never set, so it would be the default value.

Reply:
Just trying to make consistent with idiomatic Go. For example, in the case of: 
`t, err := time.Parse("20060102", "bad date")` 
The value of t is the default value